apps/nextjs/tests-e2e/tests/aila-chat/auth.test.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from "@playwright/test";
+
+import { TEST_BASE_URL } from "../../config/config";
+import { bypassVercelProtection } from "../../helpers";
+
+test.describe("Unauthenticated", () => {
+  test("redirects to /sign-in", async ({ page }) => {
+    await bypassVercelProtection(page);
+    await page.goto(`${TEST_BASE_URL}/aila`);
+    await expect(page.locator("h1")).toContainText("Sign in");
+  });
+});
```

Comment:
This is unchanged, just extracted from apps/nextjs/tests-e2e/tests/aila-chat.test.ts